src/reports/components/fast-pass-report-summary.tsx

```diff
@@ -1,16 +1,31 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as React from 'react';
-import { FastPassOutcomeSummaryBar } from 'reports/components/fast-pass-outcome-summary-bar';
+import { FastPassReportResultData } from 'reports/components/fast-pass-report';
+import { OutcomeSummaryBar } from 'reports/components/outcome-summary-bar';
+import { OutcomeStats } from 'reports/components/outcome-type';
+import { allRequirementOutcomeTypes } from 'reports/components/requirement-outcome-type';
 
-export interface FastPassReportSummaryProps {}
+export interface FastPassReportSummaryProps {
+    results: FastPassReportResultData;
+}
 
 export class FastPassReportSummary extends React.Component<FastPassReportSummaryProps> {
     public render(): JSX.Element {
+        const stats: Partial<OutcomeStats> = {
+            //hardcoded sample stats, real numbers in future PR
```

Comment:
It's a good practice to reference a specific work item rather than "future PR"; that way, if we accidentally forget to do it, it's easier for a future maintainer that notices the issue to look up context about it. Can you note the User Story number where you expect this to be implemented, like in L21 of the test?

Reply:
thank you Dan, updated!